lib/FrameManager.js

```diff
@@ -618,7 +618,7 @@ class Frame {
    * @return {!Promise}
    */
   waitForFunction(pageFunction, options = {}, ...args) {
-    const timeout = options.timeout || 30000;
+    const timeout = (options.timeout != null) ? options.timeout : 30000;
```

Comment:
nit: Let's use `helper.isNumber` instead.
```js
const timeout = helper.isNumber(options.timeout) ? options.timeout : 30000;
```

Reply:
@aslushnikov Great, do you want me to rollback the eslint config to `== null` too?

Reply:
> Great, do you want me to rollback the eslint config to == null too?

@alixaxel yes, I like keeping it as strict as possible. Thanks!